src/ui/flutter_app/lib/screens/about_page.dart

```diff
@@ -136,10 +136,18 @@ class AboutPage extends StatelessWidget {
     await launch(url);
   }
 
-  Future<void> _launchFeedback() async {
+  Future<String?> _getCurrentLocale() async {
```

Comment:
This should be converted into a getter and maybe renamed to locale
``` dart
Future<String?> get locale asnyc {
  //some stuff
}
```

Also is there a reason not to use `Localizations.localeOf(context);` the result isn't nullable so it would make things a bit easier. We also wouldn't need to await the future. 

This would actually remove the need for the helper method all together

Reply:
So what is the recommended way?  I didn't understand too much.

Reply:
I think using `Localizations.localeOf(context)` is the prefered way.